incubator/vault-token-injector/templates/deployment.yaml

```diff
@@ -37,14 +37,22 @@ spec:
           args:
             - -c
             - /config/config.yaml
+            - -v{{ .Values.logLevel }}
           volumeMounts:
           - name: config
             mountPath: /config
+          envFrom:
+          - secretRef:
+              name: {{ include "vault-token-injector.fullname" . }}
           env:
           - name: "CIRCLE_CI_TOKEN"
             value: {{ .Values.circleToken | quote }}
```

Comment:
Should we remove this since we're pulling it from a secret now?

Reply:
Good catch